Automattic/wp-super-cache
# Adds Jetpack Boost links now that Jetpack Boost is available as a standalone plugin

#### Description

This adds Jetpack Boost to the recommended plugins and side box for speeding up your website. 

### Testing instructions
- Check the link added to the yellow box takes you to the Boost homepage
- Check the link in the recommended section (bottom of settings) also takes you to the Boost homepage
